### PR TITLE
Employment pages

### PR DESCRIPTION
**Pull request by emilyville**
Employment landing content adjustment to match IA for not logged in view
move apprenticeship page that was missing out of commitments directory
delete commitments directory so it can keep living in VEC
delete commitments page so that it can combine with commitments page in VEC

Remove dummy links, links to vec, and format for usa-width character lengths as I find them.

### CODE REVIEW

File: _employment/employers/support-veteran-employees.md [hunk 1]
```diff
@@ -3,6 +3,13 @@ title: Support Veteran Employees
 concurrence: incomplete
 template: 1-topic-landing
 ---
+<div class="action-bar">
+  <div class="row">
+    <div class="small-12 columns">
```
Comment (webinista) [inline _employment/employers/support-veteran-employees.md]: @emilyville Should this also have a `usa-content` content class to contain the width of each line?

**Comment by webinista** [comment]
It looks like `post-a-job.md` isn't getting parsed correctly by Jekyll. I think it's because there's Markdown within an HTML tag and that's a bit harder for Jekyll to handle.

File: _employment/employers/support-veteran-employees.md (hunk 1 above)
Comment (emilyville) [inline _employment/employers/support-veteran-employees.md]: Sure, but lets make that a bigger change across all the places we do this? It's not how it is currently elsewhere.

**Comment by emilyville** [comment]
@webinista I am not totally following your comment. Why do you think it's not getting parsed correctly? And is it a result of this pull request?

**Comment by emilyville** [comment]
@webinista or @ayaleloehr please review and merge! if it's not looking ok ping me and I will fix. This one is also prone to merge conflicts since it's across so many files.

**Comment by ayaleloehr** [comment]
@emilyville after merging #1542, I merged job-employment-v2 into branch and it dropped this PR to only 3 files, so I think there was a lot of overlap between those PRs. I don't feel comfortable merging because it doesn't look like this addresses the goals at the top of this PR anymore. The code LGTM so if you feel this PR is still accomplishing it's objective, feel free to merge.

**Pull request merged**